pkg/v1/tkg/test/data/testdata/customClusterBootstrapAntrea.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: cni.tanzu.vmware.com/v1alpha1
+kind: AntreaConfig
+metadata:
+  name: cc20
+  namespace: shivaani-ns
+spec:
+  antrea:
+    config:
+      disableUdpTunnelOffload: false
+      featureGates:
+        AntreaPolicy: true
+        AntreaProxy: true
+        AntreaTraceflow: false
+        Egress: true
+        EndpointSlice: true
+        FlowExporter: false
+        NodePortLocal: true
+      noSNAT: false
+      trafficEncapMode: encap
+---
+apiVersion: run.tanzu.vmware.com/v1alpha3
+kind: ClusterBootstrap
+metadata:
+  annotations:
+    tkg.tanzu.vmware.com/add-missing-fields-from-tkr: v1.23.5---vmware.1-tkg.1-zshippable
+  name: cc20
+  namespace: shivaani-ns
```

Comment:
you sure you want to use "shivaani-ns" ? maybe soemthing like antrea-test-...
also you can drop the "ns" its redundant.

Reply:
Fixed